cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -174,42 +177,71 @@ define(
                             // 2.0, the latest version of jasmine-ajax (mock-ajax.js) does have the
                             // necessary support.
 
-                            _.each(
-                                [
-                                    {
-                                        desc: 'completion',
-                                        responseStatus: 204,
-                                        statusText: ActiveVideoUpload.STATUS_COMPLETED,
-                                        progressValue: 1,
-                                        presentClass: 'success',
-                                        absentClass: 'error'
-                                    },
-                                    {
-                                        desc: 'failure',
-                                        responseStatus: 500,
-                                        statusText: ActiveVideoUpload.STATUS_FAILED,
-                                        progressValue: 0,
-                                        presentClass: 'error',
-                                        absentClass: 'success'
-                                    }
-                                ],
+                            _.each([true, false],
+                                function(viewRefresh) {
+                                    var refreshDescription = viewRefresh ? ' (refreshed)' : ' (not refreshed)';
+                                    _.each(
```

Comment:
@tanmaykm I think, we are using `_.each` here many times which increases the complexity of the code. We can easily cover this in one outer `_.each([true, false],` and in its function part we set `subCases = [ ... ` and have `describe('and upload ' + subCaseInfo.desc, function() {` in the same function. this would save time from one `_.each` which is pushing values to `subCases` and nothing else.

Updated structure may look like :
```
_.each([true, false],
    function(viewRefresh) {
    var refreshDescription = viewRefresh ? ' (refreshed)' : ' (not refreshed)';
    var subCases = [
              {
                 desc: 'completion' + refreshDescription,
                 responseStatus: 204,
...
...
...
    _.each(subCases,
        function(subCaseInfo) {
            describe('and upload ' + subCases.desc, function() {
...
...
...
```